interpreter/interface.go

```diff
@@ -369,7 +369,7 @@ type ValueWalkContext interface {
 var _ ValueWalkContext = &Interpreter{}
 
 type EventContext interface {
-	EmitEvent(event *CompositeValue, eventType *sema.CompositeType, locationRange LocationRange)
+	EmitEventValue(event *CompositeValue, eventType *sema.CompositeType, locationRange LocationRange)
```

Comment:
There are two `EmitEvent` functions coming from two different interfaces now. So renamed this one.